config/routes.rb

```diff
@@ -29,6 +31,8 @@
   end
   post '/', to: github_webhooks, constraints: Hookshot.constraint
 
+  get '/teams' => 'pages#teams', as: :teams
```

Comment:
Mentioned this in another comment, but we should route `/teams` to `teams#index` and create a `TeamsController`.